draft-ietf-trans-rfc6962-bis.md

```diff
@@ -757,39 +757,61 @@ Final STH:
 [JSON.Metadata] is an example of a metadata format which includes the above
 elements.
 
-## Accepting Submissions
-
-To ensure that logged certificates and precertificates are attributable to a
-known trust anchor, and to set clear expectations for what monitors would find
-in a log, and to avoid being overloaded by invalid submissions, the log MUST NOT
-accept any submission until it has verified that the submitted certificate or
-precertificate chains to an accepted trust anchor.
-
-The log MUST NOT use other sources of intermediate CA certificates to attempt
-certification path construction; instead, it MUST only use the intermediate CA
-certificates provided in the submission, in the order provided.
-
-Logs SHOULD accept certificates and precertificates that are fully valid
-according to RFC 5280 [RFC5280] verification rules and are submitted with such
-a chain. (A log may decide, for example, to temporarily reject valid submissions
-to protect itself against denial-of-service attacks).
-
-Logs MAY accept certificates and precertificates that have expired, are not yet
-valid, have been revoked, or are otherwise not fully valid according to RFC
-5280 verification rules in order to accommodate quirks of CA
-certificate-issuing software.
-However, logs MUST reject submissions without a valid signature chain to an
-accepted trust anchor. Logs MUST also reject precertificates that do not conform
-to the requirements in {{precertificates}}.
+## Evaluating Submissions
 
-Logs SHOULD limit the length of chain they will accept. The maximum chain length
-is one of the log's parameters (see {{log_parameters}}).
+A log determines whether to accept or reject a submission by evaluating it
+against the minimum acceptance criteria (see {{minimum_criteria}}) and against
+the log's discretionary acceptance criteria (see {{discretionary_criteria}}).
+
+If the acceptance criteria are met, the log SHOULD accept the submission. (A log
+may decide, for example, to temporarily reject acceptable submissions to protect
+itself against denial-of-service attacks).
+
+### Minimum Acceptance Criteria {#minimum_criteria}
 
 The log SHALL allow retrieval of its list of accepted trust anchors (see
 {{get-anchors}}), each of which is a root or intermediate CA certificate. This
 list might usefully be the union of root certificates trusted by major browser
 vendors.
 
+To ensure that logged certificates and precertificates are attributable to an
+accepted trust anchor, and to set clear expectations for what monitors would
+find in the log, and to avoid being overloaded by invalid submissions, the log
+MUST reject a submission if any of the following conditions are not met:
+
+* The submitted certificate or precertificate, together with the zero or more
+  intermediate CA certificates provided in the submission (in the order
+  provided), MUST form a valid signature chain to an accepted trust anchor. (The
```

Comment:
Does "valid" here just mean "the signatures verify" or something else?

Reply:
Yes, "valid signature chain" is intended to just mean "the signatures verify".

I think what you're implying is that the term "valid" is probably overloaded in people's minds, and I guess I'd agree.  Shall I rework this to say something like "each cert's signature must be verifiable by the public key of the next cert in the chain"?

Reply:
That seems like a good idea.

Reply:
Addressed by https://github.com/robstradling/certificate-transparency-rfcs/commit/9665c108c20c8a1ee755ffd3d22401c343f152db.

@ekr Does that look OK?